android/app/build.gradle

```diff
@@ -375,7 +375,8 @@ dependencies {
 
     implementation 'com.squareup.okhttp3:okhttp:4.9.2'
 
-    implementation "androidx.webkit:webkit:1.9.0"
+    implementation 'androidx.webkit:webkit:1.8.0'
```

Comment:
Is it necessary to downgrade `androidx.webkit:webkit`?

Reply:
Not anymore, I gonna revert this change.